doc/guides/collaborator-guide.md

```diff
@@ -1,4 +1,4 @@
-# Node.js Collaborator Guide
+# Node.js Collaborator guide
```

Comment:
This should either follow title-style capitalisation:
```suggestion
# Node.js Collaborator Guide
```

or sentence-case capitalisation:
```suggestion
# Node.js collaborator guide
```

unless I'm missing something?

Reply:
I am considering 'Collaborator' as a _noun_
and 'guide' to be a _verb_ here.

Based on sentence case guidelines, we capitalize first characters and proper nouns.
I might be wrong. I am really confused about this one.

Reply:
_Disclaimer: I'm not a native English speaker, the following comment might be a misunderstanding due to my insufficient knowledge of the language. Apologies if that's the case._

Here's a definition of proper noun from The American Heritage® Dictionary of the English Language, 5th Edition:
> A noun denoting a particular person, place, organization, ship, animal, event, or other individual entity.

I don't think collaborator fits the definition, as it is (thankfully) not an individual entity.

Anyway, I think the safe thing to do is to keep using title-case for this heading, so we don't have to debate what is a proper noun or not :)